boards/pyportal/src/display.rs

```diff
@@ -0,0 +1,53 @@
+use super::pins::*;
+use atsamd_hal::ehal::blocking::delay::DelayMs;
+use atsamd_hal::prelude::*;
+use display_interface_parallel_gpio::{Generic8BitBus, PGPIO8BitInterface};
```

Comment:
surprised if the ili9341 crate doesn't re-export this

Reply:
unfortunately it is not re-exported from what i can tell